CMakeLists.txt

```diff
@@ -22,6 +22,16 @@ cmake_minimum_required(VERSION 3.10.2)
 
 # NONE = this project has no language toolchain requirement.
 project(Vulkan-Headers NONE)
+file(READ "include/vulkan/vulkan_core.h" ver)
+string(REGEX MATCH "#define VK_HEADER_VERSION_COMPLETE VK_MAKE_VERSION\\(([0-9]+), ([0-9]+), VK_HEADER_VERSION\\)" _ ${ver})
+set(VERSION_MAJOR "${CMAKE_MATCH_1}")
+set(VERSION_MINOR "${CMAKE_MATCH_2}")
+string(REGEX MATCH "#define VK_HEADER_VERSION ([0-9]+)" _ ${ver})
+set(VK_HEADER_VERSION "${CMAKE_MATCH_1}")
+set(VERSION_STRING "${VERSION_MAJOR}.${VERSION_MINOR}.${VK_HEADER_VERSION}")
```

Comment:
Some of the values we're setting are protected with an initial `VK_` (`VK_HEADER_VERSION`) while others are not (`VERSION_MAJOR`, `VERSION_MINOR`, `VERSION_STRING`).  Is there a reason they're not all protected with `VK_`?

Reply:
no reason at all, added the `VK_` guards